mmpose/models/keypoint_heads/temporal_head.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import torch.nn as nn
+from mmcv.cnn import build_conv_layer, normal_init
+
+from mmpose.core.evaluation import compute_similarity_transform
+from mmpose.core.post_processing import fliplr_regression_3d
+from mmpose.models.builder import build_loss
+from mmpose.models.registry import HEADS
+
+
+@HEADS.register_module()
+class TemporalHead(nn.Module):
```

Comment:
Would it be better to use a more specific name, like TemporalRegressionHead or something?

Reply:
Yes, I was not sure about the name. Do I also need to change the file name?

Reply:
TemporalRegressionHead looks good. The filename should be changed accordingly